sequences/misc.yaml

```diff
@@ -42,16 +42,28 @@ crate-effects: #Temp Null Some SHPs cannot be read by openra at this time, expec
 	Defaults: null
 		Length: *
 	dollar: #money
-	reveal-map: #reveal
+	reveal-map: reveal
 	hide-map: #shroudx
 	fpower: #firepowr
 	armor: #armor
 	stealth: # RA2 Doesn't have it
 	heal: #healall
 	nuke: #mltimisl
-	levelup: #veteran
+	levelup: veteran
 	speed: #speed
 
+crate:
+	idle:
+		Offset: 0, -12
+		UseTilesetExtension: true
+	land:
+		Offset: 0, -12
+		ShadowStart: 1
```

Comment:
It will only crash in conjunction with https://github.com/OpenRA/OpenRA/pull/10568 by the way.